src/notifications/discord.rs

```diff
@@ -46,9 +49,20 @@ pub struct DiscordWebHookBody {
 
 impl DiscordWebHookBody {
   pub fn new(event: &NotificationMessage) -> Self {
-    let config_file = config_file();
-    let config = read_config(config_file);
-    let server_name = config.name;
+    // Some contexts currently don't get passed in $NAME so fall back to reading from the config
+    // if it's missing or invalid UTF-8
+    let server_name = match env::var("NAME") {
```

Comment:
```rust
let server_name = fetch_var("NAME", config.name);
```

Reply:
Won't trying to read from a config file fail on CI though?

Reply:
you right :P my brain is a bit frazzled from work :P